src/collections/news/2022/2022-09-06-the-ultimate-list-of-opensource-cn-tools/index.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: "The Ultimate List of Open Source Cloud-Native Tools"
+subtitle: ""
+date: 2022-08-29 00:00:08 +0000
+category: Coverage
+author: Bill Doerrfeld
+thumbnail: ./tools.jpg
+darkthumbnail: ./tools.jpg
+eurl: https://containerjournal.com/features/the-ultimate-list-of-open-source-cloud-native-tools/
+published: true
+type: News
+---
+
+import { NewsWrapper } from "../../News.style.js";
+
+<NewsWrapper>
+
+<div className="test">
+
+There are so many great open source cloud-native tools for nearly everything you want to do.
+And they’re all in one place—look no further than the Cloud Native Computing Foundation (CNCF).This Linux Foundation body has become a locus of some stellar cloud-native open source projects. The CNCF now hosts an array of helpful packages, spanning container scheduling, observability, persistent storage, container runtime and other areas.
+
+Odds are the cloud-native DevOps tool you need has already been developed—it’s only a matter of finding it. In recent posts, we highlighted many CNCF tools across various areas. Below, we’ll gloss over each category from a birds-eye view. Click each headline for the full rundown, or read below for a summary of the tools in each category.
+
+<h3>Scheduling and Orchestration</h3>
+Kubernetes is the most popular container scheduling tool. It can be used to automate the deployment and management of multi-cloud applications. Other scheduling and orchestration utilities from CNCF include Crossplane, Fluid, Karmada, kube-rs, Open Cluster Management and Volcano.
+
+<h3>Observability and Analysis</h3>
+Prometheus tops the list of observability and analysis tools. The platform can be used to power your monitoring and alerting systems with fine-grained metrics and excellent querying capabilities. Other CNCF tools for observability and analysis include Jaeger, Fluentd, Thanos, Cortex and OpenTelemetry.
+
+<h3>Security and Compliance</h3>
+Open Policy Agent (OPA) can be used to unify cloud-native policies across the cloud-native stack. OPA uses a common language to express authorization policies and provides a policy engine to make authorization decisions. Other CNCF projects that deliver security-as-code include The Update Framework (TUF), Falco, Notary, cert-manager and Curiefense.
+
+<h3>CI/CD</h3>
+Argo is a suite of packages that help direct jobs on Kubernetes to aid a continuous delivery pipeline. Using Argo, developers can create multi-step custom workflows and share these workflows across a cluster. Other CI/CD tools hosted by CNCF include Flux, Brigade, Keptn, OpenGitOps and OpenKruise.
+<h3>Service Mesh Tools</h3>
```

Comment:
Add links to the /meshery and the /service-mesh-performance and the /projects/service-mesh-interface-conformance pages, please.